tests/security/ima/ima_appraisal_audit.pm

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test audit function for IMA appraisal
+# Note: This case should come after 'ima_appraisal_digital_signatures'
+# Maintainer: wnereiz <wnereiz@member.fsf.org>
+# Tags: poo#49568
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use bootloader_setup qw(add_grub_cmdline_settings replace_grub_cmdline_settings);
+use power_action_utils "power_action";
+
+sub audit_verify {
+}
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my $sample_app = '/usr/bin/yes';
+    my $sample_cmd = 'yes --version';
+
```

Comment:
There are lots of variable definitions like "my $sample_app = '/usr/bin/yes';" under ".../tests/.../ima/", they are the same.
I prefer to define them in ".../lib/$ima_common_file" but it is up to you.

Reply:
I will move them to lib/ when there are common subroutines, not only the variables. Thanks anyway.